.github/workflows/contracts.yml

```diff
@@ -147,18 +191,17 @@ jobs:
       - name: Set tools' path
         run: |
           tar -xvf tools/sc_meta.tar -C tools
-
-          mv tools/home/runner/.cargo/bin/sc-meta tools/
-          mv tools/usr/bin/wasm-opt tools/
-          mv tools/usr/lib/x86_64-linux-gnu/libbinaryen.so tools/
-          rm -rf tools/usr
-
           tar -xvf tools/mx_scenario_go.tar -C tools
-          mv tools/home/runner/.cargo/bin/mx-scenario-go tools/
-          mv tools/home/runner/.cargo/bin/libwasmer_linux_amd64.so tools/
-          mv tools/home/runner/.cargo/bin/libvmexeccapi.so tools/
-          rm -rf tools/home
 
+          for tool in sc-meta mx-scenario-go wasm-opt; do
```

Comment:
Maybe some parts of the workflow can be extracted in standalone scripts, which can be invoked / idea for future PRs.